src/main/java/com/google/api/codegen/transformer/ruby/RubySurfaceNamer.java

```diff
@@ -47,12 +52,18 @@ public RubySurfaceNamer(String packageName) {
         packageName);
   }
 
-  @Override
   /** The name of the class that implements snippets for a particular proto interface. */
+  @Override
   public String getApiSnippetsClassName(Interface interfaze) {
     return publicClassName(Name.upperCamel(interfaze.getSimpleName(), "ClientSnippets"));
   }
 
+  @Override
+  public String getPathTemplateName(
+      Interface service, SingleResourceNameConfig resourceNameConfig) {
+    return Name.from(resourceNameConfig.getEntityName(), "path", "template").toUpperUnderscore();
```

Comment:
"constant variable" is a self-contradiction. If it's a constant, it seems like wrapping the name with `inittedConstantName` might make sense. The odd thing to me is that `inittedConstantName` seems to use upper camel instead of upper underscore:

https://github.com/googleapis/toolkit/blob/master/src/main/java/com/google/api/codegen/util/ruby/RubyNameFormatter.java#L79